src/Lido/mappings/lidoConstants.ts

```diff
@@ -0,0 +1,23 @@
+import { BigInt } from "@graphprotocol/graph-ts";
+import {
+  LidoNodeOperatorsRegistryAddress,
+  LidoOracleAddress,
+  LidoTokenAddress,
+  LidoTreasuryAddress,
+} from "../../utils/constants";
+
+import { LidoToken } from "../../../generated/LidoTokenstETH/LidoToken";
+import { LidoOracle } from "../../../generated/LidoOracle/LidoOracle";
+import { LidoNodeOperatorsRegistry } from "../../../generated/LidoOracle/LidoNodeOperatorsRegistry";
+export { LidoOracleAddress, LidoTreasuryAddress, LidoTokenAddress, LidoNodeOperatorsRegistryAddress };
+export const loadLidoContract = (): LidoToken => LidoToken.bind(LidoTokenAddress);
+export const loadOracleContract = (): LidoOracle => LidoOracle.bind(LidoOracleAddress);
+export const loadNORegistryContract = (): LidoNodeOperatorsRegistry =>
+  LidoNodeOperatorsRegistry.bind(LidoNodeOperatorsRegistryAddress);
+
+// 1 ETH in WEI
```

Comment:
I don't like that file. You have some "constants" here and some in `src/utils/constants.ts`.
And it is a long way of specifying 32 ETH. Just use the end result and add a comment "32 ETH".